Science/CanadianElectricityGeneration/canadian-electricity-generation.ipynb

```diff
@@ -224,6 +224,37 @@
     "px.sunburst(df[df['Type']=='Territory'], path=['Area','Renewable','Source'], values='Proportion', title='Electricity Generation in the Territories')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Alternate Data Sources\n",
+    "\n",
+    "We can also obtain data from tables on a Wikipedia article. The data are likely more current, but may contain errrors or stations that are not yet operational.\n",
```

Comment:
"... on a Wikipedia article. The data are likely more current, but may contain ..."
might better read:
"... on a Wikipedia article. ~~The~~ This data ~~are~~ is likely more current, but may contain ..."
and is more consistent with common (perhaps contentious) usage of data as a collective noun.  
Unless it is meant that any data from Wikipedia and not this specific table. Then maybe  "These data" or "These datasets" would better convey the meaning.

Reply:
I like your wording of "These datasets" to refer to the tables in the Wikipedia article, which also helps sidestep the plural controversy. Consider it amended.